### PR TITLE
Bump `base`

### PR DESCRIPTION
**Pull request by alexander-yevsyukov**
This PR adds the dependency on `spine-validate` artifact recently introduced in `base`.

### CODE REVIEW

File: time/build.gradle.kts [hunk 1]
```diff
@@ -40,6 +40,7 @@ dependencies {
     compileOnly(AutoService.annotations)
 
     api("io.spine:spine-base:$baseVersion")
+    implementation("io.spine:spine-validate:$baseVersion")
```
Comment (armiol) [inline time/build.gradle.kts:43]: I am not sure I understand this properly.

`time` library provides its own Protobuf options. So it looks there should be an implementation of `OptionsProvider` (publishing `time_options.proto` to the extension registry) which would be auto-discovered by the mechanism established in recent `base`.

If that's not the case, I don't understand why we need this dependency in `time`. 

Let's discuss vocally if needed.

**Review by armiol — CHANGES_REQUESTED** [review]
@alexander-yevsyukov please see my comment/question.

File: time/build.gradle.kts (hunk 1 above)
Comment (alexander-yevsyukov) [reply time/build.gradle.kts:43]: You're right. I was in a rush... :)

**Review by armiol — APPROVED** [review]
LGTM.

**Pull request merged**